static/src/js/utils/getPermittedUser.js

```diff
@@ -0,0 +1,14 @@
+/**
+ *Logic to determine permittedUser based on environment
+ */
+const getPermittedUser = () => {
+  // For development environment
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
to be consistent with other apps, you'll want to use check AppConfig version instead.   We talked about writing a ticket for future to possibly move things over to using process.env.NODE_ENV